core/utils/misc.hpp

```diff
@@ -31,6 +31,37 @@
 
 namespace iresearch {
 
+template<typename T>
+class EboRef {
```

Comment:
Is it really necessary?
I think it temporary solution to avoid changes a lot of code?
Can you write a comment about that?

Also I think it will be better in separate header, for example old ebo header